styles.css

```diff
@@ -0,0 +1,4 @@
+.HDColor {
+  color:blueviolet;
+
+}
```

Comment:
Kindly solve your error linter, you can solve it by 

1. Add space after `color:` like `color: blueviolet;`.
2. Delete line 3 the empty one.
3. Add an empty line at the end of the file.

Please don't submit until you are sure that your linter is passing.

Reply:
Thank you.
I really appreciate your effort.